.github/workflows/hpu-gaudi2.yml

```diff
@@ -39,13 +39,14 @@ jobs:
     # The type of runner that the job will run on
     runs-on: [self-hosted, intel, gaudi2]
     container:
-      image: vault.habana.ai/gaudi-docker/1.15.1/ubuntu22.04/habanalabs/pytorch-installer-2.2.0:latest
+      image: vault.habana.ai/gaudi-docker/1.16.2/ubuntu22.04/habanalabs/pytorch-installer-2.2.2:latest
```

Comment:
@vshekhawat-hlab - This is just to update I assume (from 1.16.0 to 1.16.2) but doesn't fix the underlying issue yet, right?

Reply:
@loadams 
Yes, the underlying issue is still present, and the IDC team is working on it. I have updated to the latest version of the image.

In the latest patch, I enabled additional debugging logs to gather more information about firmware versions and dmesg logs. Can we run CI on latest patch again.

Reply:
@vshekhawat-hlab - the build has been re-run, anything else we can do to help update this version, even if we shut down this runner and move to a new node?

Reply:
Hi @loadams ,

I am checking with IDC on this. This will be resolved soon.

Reply:
HI @loadams ,

Can you please try the steps mentioned by Swadesh over email to upgrade Gaudi2 node with 1.16.

Reply:
Yes, I will soon, thanks!

Reply:
HI @loadams ,

Have you tried the steps provided by Swadesh over Email?

Reply:
From the latest run, I can see that the firmware driver versions in hl-smi are from 1.14.0.

`Firmware [FIT] Version   : Linux gaudi2 5.10.18-hl-gaudi2-1.14.0-fw-48.0.1-sec-7 #1 SMP PREEMPT Sun Jan 7 20:12:35 IST 2024 aarch64 GNU/Linux
Firmware [SPI] Version   : Preboot version hl-gaudi2-1.14.0-fw-48.0.1-sec-7 (Jan 07 2024 - 20:03:16)
Firmware [UBOOT] Version : U-Boot 2021.04-hl-gaudi2-1.14.0-fw-48.0.1-sec-7 (Jan 07 2024 - 20:11:55 +0200) build#: 10967
Firmware [OS] Version    : Zephyr 2.7.2-hl-gaudi2-1.14.0-fw-48.0.1-sec-7 (Jan  7 2024 - 20:03:29)`

Have you triggered the latest run after following the steps provided by Swadesh?

Reply:
Hi @vshekhawat-hlab - I haven't had time to yet, I'm hoping to do so later this week

Reply:
@loadams ,

Any update on driver-fw upgrade on Gaudi2 dev?

Reply:
@vshekhawat-hlab - this should be updated now.